includes/validation/class-amp-validation-error-taxonomy.php

```diff
@@ -279,6 +286,10 @@ public static function register() {
 		if ( is_admin() ) {
 			self::add_admin_hooks();
 		}
+
+		add_action( 'created_' . self::TAXONOMY_SLUG, [ __CLASS__, 'clear_cached_counts' ] );
+		add_action( 'edit_' . self::TAXONOMY_SLUG, [ __CLASS__, 'clear_cached_counts' ] );
+		add_action( 'delete_' . self::TAXONOMY_SLUG, [ __CLASS__, 'clear_cached_counts' ] );
```

Comment:
Should these instead be `edited_` and `deleted_`? The `edit_` and `delete_` actions are run before the term is filtered, while the past tense counterpart is run after the term cache has been cleaned.

Reply:
Note that `created_` is used above.

Reply:
I'm seeing that `edit_{$taxonomy}` also happens after the edit has been made. It's docblock says:

> Fires after a term in a specific taxonomy has been updated, but before the term cache has been cleaned.

Reply:
I don't see an action for `deleted_{$taxonomy}`.

Reply:
Yes your right. Just was wondering why the use of `created_{$taxonomy}` and not `create_{$taxonomy}`.

Reply:
I suppose we could use `created_{$taxonomy}` and `edited_{$taxonomy}` instead?

Reply:
I see that `saved_{$taxonomy}` is called when both inserting and editing, so that's an alternative.

Reply:
Oh wait that was introduced in WP 5.5, nevermind.

Reply:
See https://github.com/ampproject/amp-wp/pull/5343/commits/4fe39adde354da08c7f1a85e19d442eea17252bb, where `edited_{$taxonomy}` is instead used.